lunavl/sdk/luna_faces.py

```diff
@@ -89,6 +97,28 @@ def __init__(self, coreDetection: Face, image: VLImage, estimatorCollection: Fac
         self._glasses: Optional[Glasses] = None
         self._credibilityCheck: Optional[CredibilityCheck] = None
         self.estimatorCollection: FaceEstimatorsCollection = estimatorCollection
+        self._estimateMaskFromImage = estimateMaskFromImage
```

Comment:
For what this variable?  This class presupposes the most accurate estimations.If user want more flexibility he should be use a low level api (call `estimate directly).

Reply:
This variable is used in the mask property. Other modules, such as sdkloop, use this property rather than the `estimate` method

Reply:
It is a problem of sdkloop. user (sdkloop including) can inherited from this class and rewrite this method or realize your logic or does not use property and.

Reply:
Added subclass to estimate mask from image

Reply:
for what? It is should be  done in user code. Class VLFaceDetection realizes most accuracy estimation way. If user want a  change this he will do it in own code.

Reply:
Users of this library should have a high-level way of estimate a mask from an image. The high-level class for getting a detector is FaceDetector. Why, if we give a high-level way to get this, the user has to implement the low-level class themselves?